plugins/inputs/diskio/diskio_linux_test.go

```diff
@@ -14,6 +14,7 @@ import (
 var nullDiskInfo = []byte(`
 E:MY_PARAM_1=myval1
 E:MY_PARAM_2=myval2
+S:foo/bar/devlink foo/bar/devlink1
```

Comment:
Shouldn't this be multiple lines like:
```
S:foo/bar/devlink
S:foo/bar/devlink1
```

Reply:
Yes, my mistake. I will update accordingly.